tests/tracing/test_nodestats.py

```diff
@@ -7,19 +7,26 @@
 
 import numpy as np
 import numpy.typing as npt
+import pandas as pd
 import pytest
 from pytest_lazyfixture import lazy_fixture
 
-from topostats.tracing.nodestats import nodeStats
+# pylint: disable=import-error
+# pylint: disable=no-name-in-module
+from topostats.io import dict_almost_equal
+from topostats.tracing.nodestats import nodeStats, nodestats_image
 
 BASE_DIR = Path.cwd()
-RESOURCES = BASE_DIR / "tests" / "resources"
+GENERAL_RESOURCES = BASE_DIR / "tests" / "resources"
+DISORDERED_TRACING_RESOURCES = BASE_DIR / "tests" / "resources" / "tracing" / "disordered_tracing"
```

Comment:
Since directories are nested could use...

```suggestion
DISORDERED_TRACING_RESOURCES = GENERAL_RESOURCES / "tracing" / "disordered_tracing"
```